### ISSUE TITLE
Incorrect age in kubectl expose output

### ISSUE THREAD

**Issue by mwielgus** [kubernetes/kubernetes#12581]
```
$ kubectl expose rc monitoring-heapster-v7 --name=h2 --namespace=kube-system --port=7890

NAME      CLUSTER_IP   EXTERNAL_IP   PORT(S)    SELECTOR                      AGE
h2                     unknown       7890/TCP   k8s-app=heapster,version=v7   292y
```

**Comment by bgrant0607** [comment]
@hurf, could you please take a look?

**Comment by hurf** [comment]
@bgrant0607 - Sure, I'll work on it.

**Comment by bgrant0607** [comment]
I think it's because the object printed was generated locally and has no creationTimestamp field:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/expose.go#L208

**Comment by hurf** [comment]
I can deal with the situation when creationTimesatamp is null and report AGE as unknown. Aslo consider add the field for local object.

**Comment by JanetKuo** [comment]
Fixed by #12635.
